### DISCUSSION TITLE
Recent articles on daily.dev

### DISCUSSION THREAD

**Discussion by clevercoderjoy** [dailydotdev/daily#402] [Feature Requests]
Hey team!
This extension is great.
Recently after the update, I can not find the latest/recent articles filter.
Please re-add the feature it helps me to keep myself updated with the latest dev news.
Thanks

**Comment by idoshamun** [comment]
Hi @clevercoderjoy,

We decided to remove the recent feed with the sidebar release.
But we got many requests to put it back so we will reconsider and let you know :)

**Reply by clevercoderjoy** [reply]
wow! this is amazing!
plz do put it back!
thanks for quick response
appreciate it

**Reply by idoshamun** [reply]
We discussed the Recent feed matter and decided to add it back in an "opt-in" manner. You will be able to enable the Recent feed from the customization menu. We will announce once it's ready. Stay tuned! Thanks for your feedback.

**Reply by tshubbard** [reply]
yes please add Recent back! Great choice to make it opt-in thanks!

**Reply by idoshamun** [reply]
As you can see it's already in development. @sshanzel is working hard on it 🔥
![image](https://user-images.githubusercontent.com/1993245/149358706-cbf31d5b-dffb-492c-9fad-0dd96e587397.png)

**Reply by nimrodkra** [reply]
@clevercoderjoy @tshubbard the change is live: https://app.daily.dev/posts/e-dQ6UyIA
